server/balancer.go

```diff
@@ -52,13 +52,13 @@ func newCapacityBalancer(minRatio float64, maxRatio float64) *capacityBalancer {
 }
 
 // calculate the score, higher score region will be selected as balance from store,
-// and lower score region will be balance to store.
+// and lower score region will be balance to store. The score range is (0,100).
 // TODO: we should adjust the weight of used ratio and leader score in futher,
 // now it is a little naive.
 func (cb *capacityBalancer) score(store *storeInfo, regionCount int) float64 {
 	usedRatioScore := store.usedRatio()
 	leaderScore := store.leaderScore(regionCount)
-	score := usedRatioScore*0.6 + leaderScore*0.4
+	score := (usedRatioScore*0.6 + leaderScore*0.4) * 100
```

Comment:
Actually we may get 0 score, but it is invalid i think.
float64 can be int in our algorithm.